docs/settingUp.md

```diff
@@ -133,6 +133,10 @@ More information can be found at [this documentation](https://help.github.com/ar
    * Click `Next`.
    * Check `Use auto-import`. Ensure that `1.7` is used for the `Gradle JVM`.
    * Click `Finish`.
+1. During your import, the run configurations for the project were deleted locally. In order to restore these, open git shell and run the command
```

Comment:
"The project's run configurations may have been deleted locally when you imported the project. To restore them, ..."

Another way of restoring would be to discard the changes caused by the deletion. If your method does not create any additional changes in git, then it is acceptable.